feat(community tutorials): added a blog for community guides (#43)
ConsenSys/doc.zk-evm: README.md

See [more](https://docs-template.consensys.net/) information about using Docusaurus quickly.

> ### Local Development

Reviewer: ```suggestion
### Local development
```

    $ npm run serve

This command generates static content into the `build` directory.

### Contributing

> #### Contributing to our Docs

Reviewer: ```suggestion
#### Contributing to our docs
```

See something missing? Error in our documentation? Create an issue [here](https://github.com/ConsenSys/doc.zk-evm/issues).

Alternatively, help us improve our documentation! [Fork our repo](https://github.com/ConsenSys/doc.zk-evm/fork), create a pull request, and tag us for review!

You can find our contribution guidelines [here](https://docs-template.consensys.net/category/getting-started) and our style guide [here](https://docs-template.consensys.net/getting-started/style-guide).

Take a look at some [good first issues](https://github.com/ConsenSys/doc.zk-evm/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) to get started.

#### Contributing to Community Tutorials

If you've created more fleshed out guides and tutorials, we'd love to feature your content in our community tutorials section. [Fork our repo](https://github.com/ConsenSys/doc.zk-evm/fork), create a pull request, and tag us for review!

You can learn how to add a post under the `/blog` directory by following the Docusaurus instructions for [adding posts](https://docusaurus.io/docs/blog#adding-posts).

> #### Contributing to the ZK Glossary

Author: Is there a reason we don't want to call it the zk glossary? and glossary instead?

> Diving into zk rollups and getting stumped by the technical jargon? We've started an open source ZK glossary to define some common terms you might encounter as you dive into the L2 landscape.

Reviewer: ```suggestion
Diving into zero-knowledge rollups and getting stumped by the technical jargon? We've started an open source glossary to define some common terms you might encounter as you dive into the L2 landscape.
```
